runtime/src/accounts_db.rs

```diff
@@ -5125,26 +5125,70 @@ impl AccountsDb {
         }
     }
 
-    pub fn get_snapshot_storages(&self, snapshot_slot: Slot) -> SnapshotStorages {
-        self.storage
+    pub fn get_snapshot_storages(&self, snapshot_slot: Slot) -> (SnapshotStorages, Vec<Slot>) {
+        let mut m = Measure::start("get keys");
+        let keys = self
```

Comment:
I'm confused a bit by the name `keys`. Aren't these slots?

Reply:
We're iterating a dashmap<Slot, ...> to get the keys, which are also slots. Algorithmically and implementation-wise, this is the cost to get the list of keys (which, again, are slots). I want to replace this get keys step in a later pass, to instead get a range of possible keys from min_slot..=snapshot_slot. min_slot isn't available reliably right now, I don't think. So, I broke out the cost of getting the keys for the moment. This is only used in logging anyway (and the "get keys" in Measure::start("get keys") isn't used at all - it is just there for clarity, ironically.

Reply:
ah, the variable keys. I attached to "get keys" when seeing the github snippet. Ok, yes, again - similar idea. These are the keys we'll lookup in the DashMap later, but they are also conceptually slots. So, ok, sure. 'slots' is fine, too.

Reply:
Yea. I think of hashmap keys and pubkeys. I think using hashmap keys as a name is probably not the best name since it's so generic unless the function is very trivial.

Reply:
I renamed it to slots.